modules/monitoring-limiting-scrape-samples-in-user-defined-projects.adoc

```diff
@@ -20,6 +20,7 @@ can use the following measures to control the impact of unbound metrics attribut
 
 * Limit the number of samples that can be accepted per target scrape in user-defined projects
 * Limit the number of scraped labels, the length of label names, and the length of label values
+* Configure the intervals between consecutive scrapes and between Prometheus rule evaluations
```

Comment:
```suggestion
* Configure the intervals between consecutive scrapes and Prometheus rule evaluations
```

Reply:
Thank you for the suggestion, I am just a little afraid that would create a double meaning (it could sound like the interval is between scrapes and evaluations., instead of between each scrapes and between each evaluations)